experimental/vertx/templates/default/.appsody-config.yaml

```diff
@@ -0,0 +1,2 @@
+# config file for application development
```

Comment:
The travis build is failing because this file is now generated when creating a project. You can delete this now.